Lib/subprocess.py

```diff
@@ -1307,15 +1307,15 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
                 try:
                     exception_name, hex_errno, err_msg = (
                             errpipe_data.split(b':', 2))
+                    err_msg = err_msg.decode(errors="surrogatepass")
```

Comment:
Because that's how data in the format "exception_name:hex_errno:errmsg" is written: https://github.com/python/cpython/commit/4d078046605a53e061aedfb2694baade6d3f5483#diff-cc136486b4a8e112e64b57436a0619ebR1207